app/services/peopleforce_exporter.rb

```diff
@@ -0,0 +1,174 @@
+        {name: 'slack_username', value: Proc.new { |person| nil }},
+        {name: 'facebook_url', value: Proc.new { |person| nil }},
+        {name: 'linkedin_url', value: Proc.new { |person| person.linkedin }},
+        {name: 'skills', value: Proc.new { |person| person.skills }}
+    ]
+
+    jobs_mapping = [
+        {name: 'employee_id', value: Proc.new { |person| person.employee_id }},
+        {name: 'effective_from', value: Proc.new { |person| person.start_date }},
+        {name: 'compensation', value: Proc.new { |person| nil }},
+        {name: 'employment_type', value: Proc.new { |person| nil }},
+        {name: 'probation_policy', value: Proc.new { |person| nil }},
+        {name: 'position', value: Proc.new { |person| person.current_position }},
+        {name: 'department', value: Proc.new { |person| nil }},
+        {name: 'division', value: Proc.new { |person| nil }},
+        {name: 'location', value: Proc.new { |person| nil }},
+        {name: 'reporting_to_employee_id', value: Proc.new { |person| nil }}
+    ]
+
+    dayoffs_mapping = [
+        {name: 'employee_id', value: Proc.new { |dayoff| dayoff[:employee_id] }},
+        {name: 'date', value: Proc.new { |dayoff| dayoff[:date] }},
+        {name: 'type', value: Proc.new { |dayoff| dayoff[:type] }},
+    ]
+
+    dependents_mapping = [
+        {name: 'employee_id', value: Proc.new { |person| person.employee_id }},
+        {name: 'first_name', value: Proc.new { |person| nil }},
+        {name: 'last_name', value: Proc.new { |person| nil }},
+        {name: 'date_of_birth', value: Proc.new { |person| nil }},
+        {name: 'gender', value: Proc.new { |person| nil }},
+    ]
+
+    candidates_mapping = [
+        {name: 'first_name', value: Proc.new { |person| person.name.split(' ').first }},
+        {name: 'last_name', value: Proc.new { |person| person.name.split(' ').drop(1).join('') }},
+        {name: 'email', value: Proc.new { |person| person.personal_email }},
+        {name: 'gender', value: Proc.new { |person| nil }},
+        {name: 'skills', value: Proc.new { |person| person.skills }},
+        {name: 'phone_number', value: Proc.new { |person| person.phone }},
+        {name: 'skype_id', value: Proc.new { |person| person.skype }},
+        {name: 'source', value: Proc.new { |person| nil }},
+        {name: 'linkedin_url', value: Proc.new { |person| person.linkedin }},
+        {name: 'facebook_url', value: Proc.new { |person| nil }},
+        {name: 'added_on', value: Proc.new { |person| person.created_at }},
+        {name: 'date_of_birth', value: Proc.new { |person| person.day_of_birth }},
+        {name: 'desired_salary', value: Proc.new { |person| person.expected_salary }},
+        {name: 'currency_code', value: Proc.new { |person| nil }}
+    ]
+
+    @employees_table = []
+    @jobs_table = []
+    @dependents_table = []
+    @candidates_table = []
+    @dayoffs_table = []
+
+    Person.all.each do |person|
```

Comment:
exclude deleted

Reply:
employees are Person with employee hiring_status, Candidates the same with other statuses ..